app/src/services/converter.js

```diff
@@ -0,0 +1,69 @@
+const logger = require('logger');
+const ogr2ogr = require('ogr2ogr').default;
+const XLSX = require('xlsx');
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper');
+
+class OGRConverter {
+
+    static async convert(ctx) {
+        logger.info('Converting file...', ctx.request.body);
+        logger.debug(`[OGRRouterV2 - convertV2] request: ${JSON.stringify(ctx.request)}`);
+        ctx.assert(ctx.request.files && ctx.request.files.file, 400, 'File required');
+        logger.debug(`[OGRRouterV2 - convertV2] file data: ${JSON.stringify(ctx.request.files.file)}`);
+
+        let result;
+        logger.info(`[OGRRouterV2 - convertV2] file type: ${ctx.request.files.file.type}`);
+        logger.debug(`[OGRRouterV2 - convertV2] file size: ${ctx.request.files.file.size}`);
+
+        if (ctx.request.files.file.mimetype === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet') {
+            logger.info('[OGRRouterV2 - convertV2] It is an excel file');
+            const xslxFile = XLSX.readFile(ctx.request.files.file.filepath);
+            const csvPAth = path.parse(ctx.request.files.file.filepath);
+            csvPAth.ext = '.csv';
+            csvPAth.base = csvPAth.name + csvPAth.ext;
+            XLSX.writeFile(xslxFile, path.format(csvPAth), { type: 'file', bookType: 'csv' });
+            ctx.request.files.file.filepath = path.format(csvPAth);
+            let options = ['-t_srs', 'EPSG:4326', '-oo', 'GEOM_POSSIBLE_NAMES=*geom*', '-oo', 'HEADERS=AUTO', '-oo', 'X_POSSIBLE_NAMES=Lon*', '-oo', 'Y_POSSIBLE_NAMES=Lat*', '-oo', 'KEEP_GEOM_COLUMNS=YES'];
+            result = await ogr2ogr(ctx.request.files.file.filepath, { options, timeout:  60000});
+        } else {
+            logger.info('[OGRRouterV2 - convertV2] Not an excel file');
+
+            let options;
+            let inputPath = ctx.request.files.file.filepath
+
+            if (ctx.request.files.file.mimetype === 'text/csv' || ctx.request.files.file.mimetype === 'application/vnd.ms-excel') {
+                logger.debug('csv transforming ...');
+                // @TODO
+                options = ["-s_srs", "EPSG:4326", "-t_srs", "EPSG:4326", '-oo', 'GEOM_POSSIBLE_NAMES=*geom*', '-oo', 'CSVLINE=YES', '-oo', 'X_POSSIBLE_NAMES=lon*,Lon*', '-oo', 'Y_POSSIBLE_NAMES=lat*,Lon*', '-oo', 'KEEP_GEOM_COLUMNS=YES'];
```

Comment:
eslint is complaining about these double quotes, and more things. Please make sure eslint rules are followed

Reply:
also: I noticed some arguments are not only different, but opposite of what can be found in the original implementation. "KEEP_GEOM_COLUMNS" went from "NO" to "YES". Did you validate this causes no issues?

Reply:
In the old code `KEEP_GEOM_COLUMNS=No` was still writing out the columns in the `properties` attribute of the geojson output (maybe a bug in older versions of ogr2ogr?) so I updated that to "YES" to keep the existing behavior although I find it weird to keep the geom columns in properties.

Reply:
The other change is adding more variants of geom column names and ogr2ogr update to parse header correctly.